sdk/communication/azure-communication-chat/CHANGELOG.md

```diff
@@ -1,18 +1,17 @@
 # Release History
 
-## 1.0.0b4 (2021-02-08)
+## 1.0.0b4 (2021-02-09)
 ### Breaking Changes
-- Replaced CommunicationUser with CommunicationUserIdentifier.
-- Replaced PhoneNumber with PhoneNumberIdentifier
+- Updated to Azure.Communication.Common version 1.0.0-beta.4. Now uses `CommunicationUserIdentifier` and `CommunicationIdentifier` in place of `CommunicationUser`, and `CommunicationTokenCredential` instead of `CommunicationUserCredential`.
```

Comment:
The pacakge names for python are lower-cased, right? and the version should be 1.0.0b4 instead of 1.0.0-beta.4